tests/integration/devtools/test_callback_timing.py

```diff
@@ -0,0 +1,38 @@
+from time import sleep
+import requests
+
+import dash_html_components as html
+import dash
+from dash.dependencies import Output, Input
+
+
+def test_dvct001_callback_timing(dash_thread_server):
```

Comment:
@rpkyle this test should cover parity between Python and R for the `Server-Timing` header, including `record_timing`.